sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -536,24 +532,13 @@ object ScalaJSPluginInternal {
       }
   )
 
-  val scalaJSTestSettings = (
+  val scalaJSTestSettings: Seq[Setting[_]] = (
       scalaJSTestBuildSettings ++
-      scalaJSRunSettings ++
       scalaJSTestFrameworkSettings ++
       scalaJSTestHtmlSettings
   )
 
-  val scalaJSDefaultBuildConfigs = (
-      inConfig(Compile)(scalaJSConfigSettings) ++ // build settings for Compile
-      inConfig(Test)(scalaJSTestBuildSettings)
```

Comment:
I had a hunch that somebody requested this, but I can't find any reference in the history anymore (seems like it got introduced in [5e1f3b3](https://github.com/scala-js/scala-js/commit/5e1f3b3ddbf16ccfde6cecd655f027fa780f4f3d#diff-49772688724bf66a3c342ddc1451f277R525)). So I guess its fine if we nuke it.